playbooks/setup_jenkins_slave.yml

```diff
@@ -13,6 +13,7 @@
     # These variables are not booleans so can use the standard default filter
     jenkins_node_labels: "{{ lookup('env', 'JENKINS_NODE_LABELS')|default('single_use_slave', true) }}"
     jenkins_node_executors: "{{ lookup('env', 'JENKINS_NODE_EXECUTORS')|default('2', true) }}"
+    jenkins_port: "{{ ansible_port | default(22, true) }}"
```

Comment:
The default kind of gives it away, but we could probably indicate w/ the variable name that this is the ssh port we're connecting into.